docs/en/observability/manage-cases.asciidoc

```diff
@@ -21,6 +21,27 @@ you've previously added one, that connector displays as the default selection. O
 default setting is `No connector selected`.
 . After you've completed all of the required fields, click *Create case*.
 
+[float]
+[[add-observability-case-files]]
+=== Add files
+
+After you create a case, you can upload and manage files on the *Files* tab:
+
+[role="screenshot"]
+image::images/case-files.png[A list of files attached to a case]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
+
+The acceptable file types and sizes are affected by your {kibana-ref}/cases-settings.html[{kib} case settings].
+
+When you upload a file, a comment is added to the case activity log.
+To view images, click their name in the activity or file list.
```

Comment:
Did you mean to say images here? Or should this say files? Not sure without seeing the UI. I would suggest using singular here:

 "To view a file, click it's name in the activity or file list" (or "To view an image..." if your sentence is correct.

Reply:
Only images can be viewed directly in the app at this time. Other types can only be downloaded or deleted.

Reply:
```suggestion
To view an image, click its name in the activity or file list.
```